common/src/test/java/tachyon/collections/IndexedSetTest.java

```diff
@@ -60,7 +60,10 @@ public Object getFieldValue(Pair o) {
         return o.longValue();
       }
     };
-    mSet = new IndexedSet<Pair>(mIntIndex, mLongIndex);
+    // This warning cannot be avoided when passing generics into varargs
+    @SuppressWarnings("unchecked")
+    IndexedSet<Pair> set = new IndexedSet<Pair>(mIntIndex, mLongIndex);
+    mSet = set;
```

Comment:
@SuppressWarnings needs to be used at a declaration to apply to the smallest scope possible. If we wanted to directly set mSet we would need to @SuppressWarnings at the function scope.